pkg/cmd/cli/describe/describer.go

```diff
@@ -556,12 +557,31 @@ func (d *RouteDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 
+	endpoints, endsErr := d.kubeClient.Endpoints(namespace).Get(route.Spec.To.Name)
```

Comment:
> I would call this just err

If he also chases the service to see if its valid and get its IP, then this could end up shadowing.  I think this is fine since it's a case of "this one is special"